app/notifications/notifications-list/notifications-list.component.ts

```diff
@@ -13,12 +13,21 @@ import { NotificationService } from '../notification.service';
 })
 export class NotificationsListComponent {
   @Input() notifications: Notification[];
+  @Input() selectedStatus: string = '';
+  @Input() selectedNotificationType: string = '';
+  getFilterSortingOptions: boolean = false;
```

Comment:
All three added variables are really not needed. One can apply the pipe directly in the notifications.component.html - nothing has to be changed in the notifications-list.